lib/vmpooler.rb

```diff
@@ -33,7 +33,7 @@ def self.config(filepath='vmpooler.yaml')
     parsed_config[:config]['vm_checktime'] ||= 15
     parsed_config[:config]['vm_lifetime']  ||= 24
 
-    if parsed_config[:graphite]['server']
+    if defined? parsed_config[:graphite]['server']
```

Comment:
Do you know when `defined?` returns `nil` or `false`? Below it always returns a string, except when applied to an actually undefined variable (`k`).

``` ruby
irb(main):002:0> t = {graphite: {'server' => 'blah'}}
=> {:graphite=>{"server"=>"blah"}}
irb(main):003:0> defined? t
=> "local-variable"
irb(main):004:0> defined? k
=> nil
irb(main):005:0> defined? t[:graphite]['server']
=> "method"
irb(main):006:0> defined? t[:graphite]['asdf']
=> "method"
irb(main):007:0> defined? t[:graphite]
=> "method"
irb(main):008:0> defined? t[:graph]
=> "method"
```